waveshare-epd/epd2in13/epd2in13.go

```diff
@@ -12,8 +12,10 @@ import (
 )
 
 type Config struct {
-	Width  int16
-	Height int16
+	Width        int16
+	Height       int16
+	Rotation     uint8
+	DisplayWidth int16
```

Comment:
What does DisplayWidth mean? Can you maybe add a comment? It's not obvious what the difference is between Width/Height and DisplayWidth, or what DisplayWidth is for.

Reply:
That is an excellent question!